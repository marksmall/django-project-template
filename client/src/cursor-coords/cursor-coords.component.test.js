import React from 'react';
import { shallow } from 'enzyme';

import CursorCoords from './cursor-coords.component';

describe('CursorCoords Component', () => {
  it('renders without crashing', () => {
    const testee = shallow(<CursorCoords lat={0} lng={0} />);
    expect(testee).toBeDefined();
    expect(testee).toMatchSnapshot();
  });

  it('renders known lat/lon', () => {
    const value = 10;
    const text = `LAT: ${value}, LON: ${value}`;
    const testee = shallow(<CursorCoords lat={value} lng={value} />);

    expect(testee).toMatchSnapshot();
    expect(testee.text()).toEqual(text);
  });
});
