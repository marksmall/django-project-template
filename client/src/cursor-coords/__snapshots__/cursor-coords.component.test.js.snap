// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CursorCoords Component renders known lat/lon 1`] = `
<span
  className="coords"
  id="location"
>
  LAT: 
  10
  , LON: 
  10
</span>
`;

exports[`CursorCoords Component renders without crashing 1`] = `
<span
  className="coords"
  id="location"
>
  LAT: 
  0
  , LON: 
  0
</span>
`;
